compiler/rustc_query_system/src/dep_graph/graph.rs

```diff
@@ -699,6 +700,18 @@ impl<K: DepKind> DepGraph<K> {
             data.current.intern_node(*dep_node, current_deps, fingerprint)
         };
 
+        // We have just loaded a deserialized `DepNode` from the previous
+        // compilation session into the current one. If this was a foreign `DefId`,
+        // then we stored additional information in the incr comp cache when we
+        // initially created its fingerprint (see `DepNodeParams::to_fingerprint`)
```

Comment:
super super nit: missing a period ending the sentence after the parenthesis here, making this whole paragraph look like one big run-on sentence.